typescript/baseDAO/api-extractor.json

```diff
@@ -0,0 +1,366 @@
+// SPDX-FileCopyrightText: 2020 TQ Tezos

```

Comment:
@heitor-lassarote Please ignore this file. On second thought, I think we don't have to include it in the repo.